firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/NewReleaseFetcher.java

```diff
@@ -79,10 +80,10 @@
       return cachedCheckForNewRelease;
     }
 
-    Task<String> installationIdTask = firebaseInstallationsApi.getId();
+    Task<String> installationIdTask = firebaseInstallationsApi.get().getId();
```

Comment:
Is it guaranteed that `firebaseInstallationsApi.get()` will return non-null here? I looks OK according to this [design doc](https://docs.google.com/document/d/1cfBSrBeJLv7LWdtva3MYQRLq-ypjHA7J8_MFAAe6L30/edit#heading=h.va4bklgza3d8) but seeing this example from FirePerf concerns me: [comment](https://github.com/firebase/firebase-android-sdk/blob/6a0b405b3a283687fb39854fb0dbe0f7d7b8e49b/firebase-perf/src/main/java/com/google/firebase/perf/config/RemoteConfigManager.java#L108-L119), [null-check](https://github.com/firebase/firebase-android-sdk/blob/6a0b405b3a283687fb39854fb0dbe0f7d7b8e49b/firebase-perf/src/main/java/com/google/firebase/perf/config/RemoteConfigManager.java).

Reply:
Yea FirePerf does seem to have a null-check that we can also do. I am not entirely sure what cases it would return null. Looking at other parts of the sdk it seems like a check is a good idea. https://github.com/firebase/firebase-android-sdk/blob/6a0b405b3a283687fb39854fb0dbe0f7d7b8e49b/firebase-abt/src/main/java/com/google/firebase/abt/FirebaseABTesting.java#L340

Reply:
The design doc says you can't call `.get()` from the constructor, so I think the way you have it now is good - it's called right before we need it.

After looking at the [integration guide](https://docs.google.com/document/d/1L77viPPs5_TgS97K89YneRBthyp1L7u7E0MquArjU4Q/edit#heading=h.dxjfnxhb6bea) I actually don't think we need the null check. Should we ask the ACore folks though?